test/template/test_template.py

```diff
@@ -153,21 +154,66 @@ def test_template_nominal_csv(tmp_path):
     tmpl = Template(format="A4", title="Sample Invoice")
     tmpl.parse_csv(HERE / "mycsvfile.csv", delimiter=";")
     tmpl.add_page()
+    tmpl["empty_fields"] = "empty"
```

Comment:
Why this change?

Reply:
So the existence of the field becomes verifiable in the document.
We're testing template parsing here, with an empty text field in the definition.

Of course there could be another test showing that a text field without content doesn't actually get rendered...

Reply:
As you see best